lib/src/selection.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021-2023 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// selection.dart
+// Definition for select
+
```

Comment:
Remember to update/fix the headers (there's an extra empty line here, update your name, copyright 2023 only, etc.

Reply:
done